### DISCUSSION TITLE
[Feature Request]: Make text selectable in Log window

### DISCUSSION THREAD

**Discussion by piperone** [beyondcode/herd-community#346] [Ideas]
### Feature Description

When reporting bugs and problems, the log window for PHP, Nginx and DNSMasq is readily available from the menubar icon. However, it would also be helpful if it was possible to select and copy text from the log window, something I'm not able to do with the log window currently.

### Is this feature valuable for other users as well and why?

Posting log output to GitHub in order to get help seems to be a reasonable use case to accomodate. Being able to select text and copy it from the log-window would help in that regard.

**Comment by sschlein** [comment]
Which log window do you actually mean? I've tried the Logs window from the Pro version and that works well, so which one doesn't?

**Comment by piperone** [comment]
I'm talking about the window that appears when clicking these buttons:
![btn](https://github.com/beyondcode/herd-community/assets/351744/6654f36e-8d47-4f7e-b2df-0b09c4de47f7)

Here's the window itself:
![window](https://github.com/beyondcode/herd-community/assets/351744/5ad318ca-7e9a-4b66-a46a-c6eacb65869d)

I don't have the pro-version, so I don't know if that makes a difference.

**Comment by sschlein** [comment]
That makes sense, not sure why it's not.

We'll add that in one of the next updates.
